unoji/Model/WashTimeModel.swift

```diff
@@ -6,36 +6,52 @@
 //
 
 import Foundation
+import RealmSwift
 
-class WashTimeModel {
-    var washTime: Date? // 流すアクションをした時刻を記録
-    // 前回流すアクションをしてから現在までの経過時間
-    var elapsedTime: Int {
-        guard let washTime = washTime else {
-            return 0 }
-        return Int(abs(washTime.timeIntervalSinceNow))
+class WashTimeModel: Object {
+    @objc dynamic var washTime: Date? // 流すアクションをした時刻を記録
+    @objc dynamic var washDay: String = ""// FSCalender用に年月日を文字列に
+    @objc dynamic var elapsedTime: Int = 0// 経過時間
+    @objc dynamic var displayIllustration: String = "baby"// 経過時間に応じたイラスト名を文字列で
+}
+
+extension WashTimeModel {
+    // 最後にRealmに記録されたデータを元にelapsedTimeとdisplayIllustrationの値を更新する
+    func setValue(lastTime: Date) {
```

Comment:
[nits]
メソッド名はコメントを見なくても中身が類推できるように命名することが一般的です。
Swiftの場合は、メソッド名と引数名が使えるので、以下のようにすると
lastTimeによって関連する属性を更新メソッドであることが分かります。

```suggestion
    func update(by lastTime: Date) {
```